services/categoryService.js

```diff
@@ -0,0 +1,9 @@
+const categoryDao = require("../models/categoryDao");
+
+const getCategoryDetail = async () => {
+  const getCategoryDetail = await categoryDao.getCategoryDetail();
+  return getCategoryDetail;
+};
+module.exports = {
+  getCategoryDetail,
+};
```

Comment:
```suggestion
const getCategoryDetail = async () => {
  return categoryDao.getCategoryDetail();
};

module.exports = {
  getCategoryDetail,
};
```
1. 작성해주신 코드 이외에 별다른 로직이 들어가지 않는다면 변수 선언과 비동기처리 없이 곧바로 return 해주시는것이 보다 더 깔끔한 코드가 될 수 있습니다!
2. `module.exports` 부분처럼 적절한 곳에 줄바꿈을 줄 수 있도록 계속 노력하고 연습하다 보면 이후에 개발자로 나아가는데 있어서 가독성 좋은 코드 작성에 큰 도움이 될 수 있습니다 :)